table_foosball/database_test.rb

```diff
@@ -0,0 +1,79 @@
+require_relative './database'
+require_relative './table_foosball_01'
+require 'minitest/autorun'
+
+class DatabaseTest < Minitest::Test
+	def setup
+		File.open("./games_test.sdb", "w") {}
+		File.open("./players_test.sdb", "w") {}
+
+		@gamedb = Database.new(:games_test)
+		@playerdb = Database.new(:players_test)
+
+		@sebastian = Player.new("Sebastian")
+		@simon = Player.new("Simon")
+		@daniel = Player.new("Daniel")
+		@kenichi = Player.new("Kenichi")
+
+		@game1 = Game.new([@sebastian, @simon, @daniel], [@kenichi], 10, 0)
+		@game2 = Game.new([@simon], [@daniel], 6, 10)
+		@game3 = Game.new([@sebastian, @simon], [@daniel, @kenichi], 10, 5)
+		@game4 = Game.new([@sebastian], [@daniel, @kenichi], 10, 9)
+
+		@players = [@sebastian, @simon, @daniel, @kenichi]
+		@games = [@game1, @game2, @game3, @game4]
+	end
+
+	def teardown
+		Game.games = nil
+		File.delete("./games_test.sdb", "./players_test.sdb")
+	end
+
+	def test_create_new_database
+		name = :test_database
+		test_database = Database.new(name)
+		
+		assert_equal Database, test_database.class
+		assert_equal name, test_database.instance_eval { @name }
+		assert_equal "./#{name}.sdb", test_database.instance_eval { @filename }
```

Comment:
This is pretty sketchy. In general, in Ruby the tests tend to be too tightly coupled to the code because you can do so much magic. This magic is something to be weary of as much in tests as in code. If you write test code you would hesitate to write outside of tests, it's usually a good smell that something wrong is going on.